scripts/install-nix-from-closure.sh

```diff
@@ -65,6 +65,21 @@ elif [ "x${1:-}" != "x" ]; then
     exit
 fi
 
+if [ "$(uname -s)" = "Darwin" ]; then
+    canondest="/opt/nix"
+    if ! grep -q '^nix' /etc/synthetic.conf 2> /dev/null; then
+        echo "configuring /nix in synthetic.conf" >&2
+        printf "nix\t%s\n" "$canondest" | sudo tee -a /etc/synthetic.conf > /dev/null
```

Comment:
We're not quoting `$canondest` down below. Let's try to be consistent about this; if the variable is safe to leave unquoted, let's unquote it everywhere. Otherwise the quoting here implies that maybe it should be quoted a few lines down too.